dts/arm/st/g4/stm32g47X.dtsi

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright (c) 2019 STMicroelectronics.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <st/g4/stm32g4.dtsi>
```

Comment:
Last blocking point:
I still consider this file addition is not requested.
If required, as done on L4 series, you can overwrite dma1 and dma2 property easily without any file addition.

Reply:
@lowlander if these diverge too much, let's put dma1/2 nodes definitions in stm32g431.dtsi and stm32g471.dtsi.
But I don't see a need for new intermediate files

Reply:
@erwango is the dts OK like it is now ?